bpf/lib/lb.h

```diff
@@ -31,7 +31,16 @@ struct bpf_elf_map __section_maps LB6_SERVICES_MAP_V2 = {
 
 struct bpf_elf_map __section_maps LB6_BACKEND_MAP = {
 	.type           = BPF_MAP_TYPE_HASH,
-	.size_key       = sizeof(__u16),
+	.size_key       = sizeof(__u32),
+	.size_value     = sizeof(struct lb6_backend),
+	.pinning        = PIN_GLOBAL_NS,
+	.max_elem       = CILIUM_LB_MAP_MAX_ENTRIES,
+	.flags          = CONDITIONAL_PREALLOC,
+};
```

Comment:
Why do we still need the old map? I thought the goal was to switch maps in this PR.

Reply:
> We usually just delete the maps after 3 releases, once nobody is supposed to be running a version with the old map.

Maybe I misunderstood you?

Reply:
I'm saying we get rid of v1 now. It's the migration code that we can't remove until after 3 releases.

Reply:
Makes sense, removed v1 map.